### PR TITLE
client: ability to find tasks by comment

### PR DESCRIPTION
**Pull request by rhyw**
Resolves: https://issues.redhat.com/browse/OSH-304
Closes: https://github.com/openscanhub/openscanhub/pull/224

### CODE REVIEW

File: osh/client/completion/_osh-cli [hunk 1]
```diff
@@ -75,8 +75,9 @@ _osh-cli_args()
                 '(-l --latest)'{-l,--latest}'[display only latest task]'
                 '(-r --regex)'{-r,--regex}'[query by regular expression (python, module: re)]'
                 '(-p --pacakge)'{-p,--package}'[query by package name]'
```
Comment (lzaoral) [inline osh/client/completion/_osh-cli:77]: Off topic note to self: Fix the `--pacakge` typo in a separate PR.

Comment (lzaoral) [reply osh/client/completion/_osh-cli:77]: Thank you!

**Review by lzaoral — APPROVED** [review]
LGTM, @rhyw!  I've successfully tested the feature locally.

nit: The commit message does not contain a verb which is a good practise.  For example `add ability ...`, `implement ability ...`,  etc.  Otherwise, it is not immediately obvious what was the change about (was it removed?, refactored?, ...) unless the developer inspects the diff directly.

Lets wait for @kdudka's review as well.

File: osh/client/commands/cmd_find_tasks.py [hunk 2]
```diff
@@ -42,20 +42,28 @@ def options(self):
...
+            help="query by comment"
+        )
         self.parser.add_option(
             "-n",
             "--nvr",
             default=False,
             action="store_true",
             help="query by NVR (default one)"
         )
+
```
[the remark this answers is not in the document]
Comment (rhyw) [reply osh/client/commands/cmd_find_tasks.py:59]: Im adding to split between required query and optional `--state`.

**Review by kdudka — APPROVED** [review]
Looks good.  Thanks!

**Comment by rhyw** [comment]
Thanks for the review.

**Pull request merged**